packages/core/core/src/serializer.js

```diff
@@ -119,9 +119,10 @@ function mapObject(object: any, fn: (val: any) => any, preOrder = false): any {
         if (result instanceof Map) {
           result.set(key, newValue);
         } else if (result instanceof Set) {
+          let _result = result; // For Flow
```

Comment:
This feels like a Flow bug? Why does introducing a new let binding resolve this?